packaging/aznfs/DEBIAN/control

```diff
@@ -6,4 +6,4 @@ Origin: Microsoft
 Maintainer: Azure Storage XNFSv3 IDC Team <xnfsv3devsidc@microsoft.com>
 Homepage: https://github.com/Azure/AZNFS-mount/blob/main/README.md
 Description: Mount helper program for correctly handling endpoint IP address changes for Azure Blob NFS mounts
-Depends: bash, procps, conntrack, iptables, bind9-host, iproute2, util-linux, nfs-common, netcat, sysvinit-utils
+Depends: bash, procps, conntrack, iptables, bind9-host, iproute2, util-linux, nfs-common, netcat, sysvinit-utils, dialog
```

Comment:
let's use whiptail in place of dialog. dialog has problems rendering on some terminals, it shows some characters which look ugly.
whiptail is much neater and it's default installed on most distros.
See comments below.